data/repository/build.gradle.kts

```diff
@@ -46,37 +46,45 @@ kotlin {
                 implementation(projects.data.network)
                 implementation(projects.data.persistence)
                 implementation(projects.domain.authentication)
+                implementation(projects.domain.timeline)
                 implementation(libs.io.insert.koin.core)
                 implementation(libs.kotlinx.coroutines.core)
+                //temp until we map to UI models
+                api(libs.store)
+                implementation(libs.com.squareup.sqldelight.coroutines.extensions)
             }
         }
 
 
         // android
         getByName("androidMain") {
             dependsOn(commonMain)
-            dependencies {}
+            dependencies {
+                api ("org.jetbrains.kotlinx:atomicfu:0.18.5")
```

Comment:
what's this for? can you add comment and/or add to version catalog?

Reply:
yup yup will add to catalog. It's a kotlin issue where android gets wrong version and gets a class def not found https://bytemeta.vip/repo/Kotlin/kotlinx.atomicfu/issues/145